### DISCUSSION TITLE
Patches mit Änderungen, die nicht primär mit dem eigentlichen Change zu tun haben

### DISCUSSION THREAD

**Discussion by IT-VBFK** [OpenCCU/OpenCCU#2045] [🇩🇪 Fragen&Antworten] [answered]
Hallo

Aus aktuellem Anlass ergibt sich mir die Frage: Warum ist es "schlecht" unsauberen Code (wenn auch geringfügig) zu verbessern?

Ich als Softwareentwickler bin es gewohnt, Code aufzuräumen, zu verbessern, Tippfehler zu bereinigen und solche Dinge.

Warum ist das hier nicht erwünscht, bzw. welchen Hintergrund hat das?

Vielleicht kann mich hier jemand aufklären, wäre sehr dankbar :)

**Comment by jens-maus** [comment, marked as answer]
Weil wir die CCU bzw. die WebUI nicht völlig unabhängig alleine entwickeln, sondern von eq3 bzw. deren zusätzlichen Änderungen abhängig sind. Und wenn wir jetzt sämtliche Zeilen ändern/anpassen nur weil uns was nicht gefällt werden wir in Merge-Konflikten zwischen unseren Änderungen und den Änderungen die eQ3 ggf. in der nächsten CCU Firmware Version mitbringt ersticken.

**Reply by IT-VBFK** [reply]
Ahh.. Danke :)

Das macht absolut Sinn
